pkg/state/protobuf/server/server.go

```diff
@@ -75,6 +75,17 @@ func (server *State) List(req *v1alpha1.ListRequest, srv v1alpha1.State_ListServ
 				opts = append(opts, state.WithLabelQuery(labelOpts...))
 			}
 		}
+
+		if req.Options.GetIdQuery() != nil {
+			var idOpts []resource.IDQueryOption
+
+			idOpts, err := ConvertIDQuery(req.Options.GetIdQuery())
```

Comment:
This variant and the one below are different. Is there any reason for this?

Reply:
I guess I was refactoring it and missed it, not real difference, I will clean it up, thanks